elastic/kibana
# [Security solution] Expandable flyout - add subtab to path

## Summary

This PR updates flyout path to take parameters such as `tab` and `subTab`. This allows finer navigation to the sub-tabs in the left section of expandable flyout. Users are now able to click titles in subsections under `Insights` in the right section of expandable flyout, and then be navigated to the correct tabs in the left section.

Currently available navigations to sub-tab:
- Visualization (right):
   - Session view preview -> Visualize tab (left) - Session View
   - Analyzer preview -> Visualize tab (left) - Analyzer Graph
- Insights (right)
   - Entities -> Insights tab (left) - Entities
   - Threat intelligence -> Insights tab (left) - Threat intelligence
   - Prevalence -> Insights tab (left) - Prevalence 
   - Correlation -> Insights tab (left) - Correlation
- Investigation (right)
   - Highlighted fields, host.name and user.name -> Insights tab (left) - Entities
 - Response
   - Response button -> Response tab
   
![image](https://github.com/elastic/kibana/assets/18648970/d661b011-f54d-44d3-97db-4aeb790e560c)

**How to Test**
- add `xpack.securitySolution.enableExperimental: ['securityFlyoutEnabled']` to the `kibana.dev.json` file
- go to the Alerts page, and click on the expand detail button on any row of the table
- click on `Overview`, `Insights`, click on any section headers ('Threat intelligence`), left section of flyout should be opened and the proper sub tab should be displayed.

### Checklist
